File: Jun-20-2024/enum.java
public class Main
{
    enum TournResult {
        DNQ,
        GS,
        RO16,
        QF,
        SF,
        F,
        W;
    };
    
	public static void main(String[] args) {
		System.out.println("Hello World");
		TournResult result = TournResult.DNQ;
        System.out.println(result);          // Output: DNQ
        System.out.println(result.name());   // Output: DNQ
        System.out.println(result.ordinal()); // Output: 0

        TournResult result2 = result + 1;
        System.out.println(result2);          // Output: DNQ
        System.out.println(result2.name());   // Output: DNQ
        System.out.println(result2.ordinal()); // Output: 0
	}
}

File: Jun-20-2024/enum.py
from enum import Enum


class TournResult(Enum):
    DNQ = 0
    GS = 1
    RO16 = 2
    QF = 3
    SF = 4
    F = 5
    W = 6


# Example usage
result = TournResult.DNQ
print(result)          # Output: GameResult.DNQ
print(result.name)     # Output: DNQ
print(result.value)    # Output: 0
